usmqe_tests/api/grafana/test_brick_dashboard.py

```diff
@@ -0,0 +1,98 @@
+"""
+REST API test suite - Grafana dashboard brick-dashboard
+"""
+
+import pytest
+from usmqe.api.grafanaapi import grafanaapi
+
+
+LOGGER = pytest.get_logger('brick_dashboard', module=True)
+"""@pylatest default
+Setup
+=====
+
+Prepare USM cluster accordingly to documentation.
+``GRAFANA`` for this file stands for Grafana API url used by tested Tendrl
+server.
+``GRAPHITE`` for this file stands for Graphite API url used by tested Tendrl
+server.
+
+"""
+
+"""@pylatest default
+Teardown
+========
+"""
+
+
+def test_layout():
```

Comment:
Could you rename this to something like `test_brick_dashboard_layout`? It would make logs more easier to read.

Reply:
(I'm asking Filip for the similar change in the other pull request https://github.com/usmqe/usmqe-tests/pull/172)